src/components/StatsCard/StatsCard.tsx

```diff
@@ -0,0 +1,41 @@
+const statsCardStyle = 'card bg-base-100 w-full px-3 py-2 h-28 md:h-full';
+
+type StatsCardProps = {
+  title: string;
+  value: string;
+  auxiliaryStat?: {
+    title: string;
+    colorAttribute?: ColorAttribute;
+  };
+};
+
+type ColorAttribute = 'positive' | 'negative' | 'neutral' | 'other';
```

Comment:
If you're okay having 'other', it might make more sense to so it like this:

```suggestion
type ColorAttribute = 'positive' | 'negative' | string;
```

And have the style you were going to use for 'neutral' be your default style.